src/core/public/public.api.md

```diff
@@ -450,106 +450,221 @@ export const DEFAULT_APP_CATEGORIES: Record<string, AppCategory>;
-            readonly kueryQuerySyntax: string;
-        };
-        readonly date: {
-            readonly dateMath: string;
-        };
-        readonly management: Record<string, string>;
-        readonly visualize: Record<string, string>;
+        readonly boolQuery: string;
+      };
+      readonly searchTemplate: {
+        readonly base: string;
+        readonly create: string;
+        readonly execute: string;
+        readonly advanced_operation: string;
+        readonly multiple_search: string;
+        readonly manage: string;
+      };
+      readonly searchExperience: {
+        readonly base: string;
+        readonly autocomplete: string;
+        readonly paginate: string;
+        readonly scroll: string;
+        readonly sort: string;
+        readonly highlight_match: string;
+      };
+      readonly logs: {
+        readonly base: string;
+        readonly application_log: string;
+        readonly slow_log: string;
+        readonly deprecation_log: string;
+      };
+      readonly snapshotRestore: {
+        readonly base: string;
+        readonly register: string;
+        readonly take_snapshot: string;
+        readonly restore_snapshot: string;
+        readonly security_plugin: string;
+      };
+      readonly supportedUnits: string;
+      readonly commonParameters: string;
+      readonly popularAPI: string;
+      readonly restAPI:string;
     };
-    // (undocumented)
-    readonly OPENSEARCH_WEBSITE_URL: string;
+    readonly opensearchDashboards: {
+      readonly introduction: string;
+      readonly installation: {
+        readonly base: string;
+        readonly docker: string;
+        readonly tar: string;
+        readonly helm: string;
+        readonly tls: string;
+        readonly plugins: string;
+      };
+      readonly mapTiles: string;
+      readonly ganttCharts: string;
+      readonly reporting: string;
+      readonly notebooks: {
+        readonly base: string;
+        readonly notebook_tutorial: string;
+        readonly paragraph_tutorial: string;
+        readonly sample_notebook: string;
+        readonly create_report: string;
+      };
+      readonly dql: {
+        readonly base: string;
+        readonly terms_query: string;
+        readonly boolean_query: string;
+        readonly date_query: string;
+        readonly nested_query: string;
+      };
+    };
+    noDocumentation: {
+      readonly auditbeat: string;
+      readonly filebeat: string;
+      readonly metricbeat: string;
+      readonly heartbeat: string;
+      readonly logstash: string;
+      readonly functionbeat: string;
+      readonly winlogbeat: string;
+      readonly siem: string;
+      readonly indexPatterns: {
+        readonly loadingData: string;
+        readonly introduction: string;
+      };
+      readonly management: Record<string, string>;
+      readonly visualize: Record<string, string>;
+      readonly addData: string;
+    };
+    readonly scriptedFields: {
```

Comment:
Doesn't this exist inside of `noDocumentation`?

Reply:
Nice catch.  I was changing the structure in `doc_links_service` but forgot to copy paste here